include/openthread/srp_client.h

```diff
@@ -515,15 +515,27 @@ const otSrpClientService *otSrpClientGetServices(otInstance *aInstance);
  * that the server holds the host name in reserve for when the client is once again able to provide and register its
  * service(s).
  *
- * @param[in] aInstance        A pointer to the OpenThread instance.
- * @param[in] aRemoveKeyLease  A boolean indicating whether or not the host key lease should also be removed.
+ * The @p aSendUnregToSever determines the behavior when the host info is not yet registered with the server. If
```

Comment:
Is aSendUnregToSever a typo of aSendUnregToServer

Reply:
Changed it in new push. Thanks.

Reply:
The typo exists in the PR description as well.

Reply:
fixed everywhere now. thanks.